lib/hanami/router/node.rb

```diff
@@ -34,29 +31,34 @@ def put(segment)
 
       # @api private
       # @since 2.0.0
-      def get(segment)
-        @fixed&.fetch(segment, nil) || @variable
+      def get(segment, param_values)
+        fixed = @fixed&.fetch(segment, nil)
```

Comment:
```suggestion
        fixed = @fixed&.[segment]
```
Might be faster?

Reply:
Interesting. I think that was the original style, probably just to take advantage of the safe navigator. I hadn't thought of your alternative. I'll try it.

Reply:
Okay, this appears to be a syntax error. I guess this is one place where syntactic sugar is required.

Reply:
Ah oops! My bad, I should have checked.